pandas/core/generic.py

```
@@ -6340,10 +6340,12 @@ def astype(
             mapping, e.g. {col: dtype, ...}, where col is a column label and dtype is
             a numpy.dtype or Python type to cast one or more of the DataFrame's
             columns to column-specific types.
-        copy : bool, default False
-            Return a copy when ``copy=True`` (be very careful setting
-            ``copy=False`` as changes to values then may propagate to other
-            pandas objects).
+        copy : bool or None, default None
+            Whether to copy the data or not.
+
+            - True : always return a new object.
+            - False : try to avoid copying and return a view if possible.
```

With copy-on-write now the default, this seems misleading to me.